octobot_trading/exchanges/connectors/exchange_simulator.py

```diff
@@ -110,6 +110,16 @@ def get_available_time_frames(self):
                     for time_frame in backtesting_api.get_available_time_frames(next(iter(self.exchange_importers)))]
         return []
 
+    def get_backtesting_data_files(self):
```

Comment:
Why a simulated exchange should have those two methods ? (`get_backtesting_data_files` and `get_backtesting_data_file`)